### DISCUSSION TITLE
Availability of the example datasets

### DISCUSSION THREAD

**Discussion by KarolienHeymans** [rajewsky-lab/openst#10] [Q&A] [answered]
Hello,
I'm interested in following the tutorial, but the links to access the raw data are not working. Could you please let me know when the datasets will be made available? Thank you!

**Comment by gianmore** [comment]
Hi, same here. Thanks!

**Comment by danilexn** [comment]
Hi! Thanks so much for your messages! We have uploaded all data to GEO, and will be made available once the peer-reviewed publication is accepted. In the meantime, please let us know by email, and we can provide the raw data and/or processed h5ad files.

Let us know if this works for you :hugs:

**Reply by KarolienHeymans** [reply]
Awesome! Thank you :)

**Reply by gianmore** [reply]
Thanks! Can you share to which email shall we contact you?

**Comment by gianmore** [comment]
Hi,

Thank you. It will great if you could provide me both the raw and processed
data for the mouse adult hippocampus so that I can follow along your
tutorial. Thanks!

Best,

Gianluca

**Comment by danilexn** [comment, marked as answer]
@gianmore @KarolienHeymans see #16

**Reply by gianmore** [reply]
Thank you!!

**Reply by KarolienHeymans** [reply]
Thank you so much!

**Reply by KarolienHeymans** [reply]
Hello again! I couldn't locate an h5ad file name in the tutorial that corresponds to the one in the datasets. I'm curious to know at which stage this file was generated. Thank you!

**Reply by danilexn** [reply]
Hello! I will upload these h5ad files later today, sorry for the inconvenience!

**Reply by KarolienHeymans** [reply]
That would be very helpful, thank you.

**Comment by StevenSong** [comment]
would it be possible to share the fastq files from the barcode registration sequencing step?

**Reply by nukappa** [reply]
Hi @StevenSong, yes, we will add these files in the next days.

**Reply by danilexn** [reply]
Hi @StevenSong!

We added links to download fastq files from the spatial barcode registration step - subsets of fc_1 and fc_2 that are relevant for the E13 and adult mouse samples. Enjoy :wink: 

https://rajewsky-lab.github.io/openst/latest/examples/datasets/

**Reply by StevenSong** [reply]
Perfect, thank you both

**Reply by gianmore** [reply]
Hi, I have maybe a stupid question. 
During the configuration step of spacemake, when you add the specie, mouse in this case, which rRNA sequence have you used? --sequence <path_to_rRNA_sequences.fa>
Is it mandatory to have one? 
Same for --sequence <path_to_phiX_genome.fa>

Thanks in advance!

**Reply by danilexn** [reply]
Hi! 

It is not necessary, indeed. 

With spacemake, you can specify [mapping strategies](https://spacemake.readthedocs.io/en/latest/projects/index.html#sample-parameters) (see `map_strategy` under "Sample parameters"). For mapping with genome only, you can specify a map_strategy like `STAR:genome:final`. 

Hope this helps!

**Reply by gianmore** [reply]
it helps a lot, thanks! If I would like to use the rRNA also, from where could I download it? Thanks!
